### ISSUE TITLE
Provide a style for lists of input parameters

### ISSUE THREAD

**Issue by jyasskin** [w3c/tr-design#130]
At the moment, I'm using:

```html
<dl>
 <dt><var>rpId</var>
 <dd>
  <p>The caller’s RP ID, as determined by the user agent and the client.</p>
 <dt><var>hash</var>
 <dd>
  <p>The hash of the serialized client data, provided by the client.
 ...
</dl>
```

Which renders as 
![Input parameter rendering](https://user-images.githubusercontent.com/83420/31146771-83a3c928-a83c-11e7-9fc0-0594fe252228.png)
and uses more vertical space than I want. The bold variable names also aren't great.

I don't have a strong opinion on what the style should be, as long as the experts here have thought about it.

**Comment by marcoscaceres** [comment]
Maybe we need a `input-params` class that can be applied to `dl`?... similarly to what we have with `<ol class=algorithm>`

**Comment by fantasai** [comment]
Depending on the complexity of the input parameter descriptions, you may or may not want a compact design. :) Your list looks perfectly fine to me, fwiw, so I can't quite agree that there's something to fix here. :/

@marcoscaceres Also not convinced `input-params` is a legit class name if the consideration is “I would like a different styling because my preferences are different” rather than “I need to distinguish this type of thing from the other type of thing.“

Also that list should be using `<dfn>` in the `<dt>`s, no? Are these not the defining instances of those terms?

**Comment by fantasai** [comment]
(Note that we do want visual consistency across our specs, so while I'm happy to adjust styles across W3C to get closer to an optimum overall, and to create classes where there's an established pattern of requiring visual distinction, customizing your spec's styling to more closely match your preferences when they might conflict with others isn't something I can do.)

**Comment by jyasskin** [comment]
I'd much rather have cross-spec consistency than customize my spec's styling to match my preferences. If you believe the plain `<dl>` is the right semantic way to mark this up, I won't complain.

Those `<dt>`s are the defining instance of those variable names, but Bikeshed (@tabatkins) doesn't encourage us to use `<dfn>` for defining variables (https://tabatkins.github.io/bikeshed/#var-and-algorithms), and I believe it'll do the wrong thing by giving the `<dfn>` a scope that reaches way outside of the actual scope of the variable (https://tabatkins.github.io/bikeshed/#definitions).

Maybe the right markup is to mark the `<dfn>` as having a constrained scope instead of the `<dl>` as containing things with that constrained scope. I'm a little sad about that because it'll lead to more keystrokes, but I'll get over it.

**Comment by tabatkins** [comment]
> but Bikeshed (@tabatkins) doesn't encourage us to use `dfn` for defining variables

Right, `<dfn>` generally implies a universal scope. (You can have document-local dfns (give them an explicitly empty `lt` and use `local-lt` instead), but they're still document-scoped, not algorithm-scoped.)

The long-term plan for vars is to give them defining instances as well, tho, so algorithms and their inputs become first-class autolinking objects. This has no bearing on the styling, of course.  

(And maybe the markup will switch to `dfn`, I dunno.)

**Comment by fantasai** [comment]
If it's the “defining instance” of a variable, it does seem to me that `<dfn>` is the correct markup. There's likely cases where we don't want to bother, but it's certainly not wrong...
